dataeng/resources/vault-config.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
I would add some documentation in this script to explain why we are doing this, not just what we are doing. It was difficult to deduce what was causing our errors in the first place, and having docs to explain our reasoning will be useful for anyone who needs to use it in the future.